baybe/recommenders/pure/nonpredictive/base.py

```diff
@@ -39,9 +41,21 @@ def recommend(  # noqa: D102
                 f"consider any objectives, meaning that the argument is ignored.",
                 UnusedObjectWarning,
             )
+        if (pending_experiments is not None) and (
+            self.allow_recommending_pending_experiments
```

Comment:
I'm not sure if I get the logic here.... Why is `self.allow_recommending_pending_experiments` part of the condition?

Reply:
if the flag is on, the warning doesnt make sense to me
If the user wants to allow pending recs, why would we warn them that the algo cant take into consideration pending points?

Reply:
OK, guess it makes sense. Was rather me not parsing the boolean expression correctly in my first attempt 🙃